bake/bake.go

```diff
@@ -42,14 +44,13 @@ type Override struct {
 }
 
 func defaultFilenames() []string {
-	return []string{
-		"docker-compose.yml",  // support app
-		"docker-compose.yaml", // support app
+	composeNames := composecli.DefaultFileNames
+	return append(composeNames, []string{
```

Comment:
:eyes: will this end up modifying the contents of `composecli.DefaultFileNames`? I *think* the slices will end up sharing the same storage?

I think this would be the fix?

```go
composeNames := append([]string, composecli.DefaultFileNames)
```

Reply:
It will change the underlying storage but the original `DefaultFileNames` var will still only point to the original slice contents. Probably worth changing though just to eliminate any confusion.